### PR TITLE
Add info about launching other activities

### PR DESCRIPTION
**Pull request by imurchie**
It is possible to launch activities other than those in the app the tests are associated with.

### CODE REVIEW

File: research.md [hunk 1]
```diff
@@ -188,3 +188,48 @@ How to run an espresso testcase from the command line:
...
+## Instrumenting an app that is not the one the tests are associated with
+
+### App under test (AUT)
+* Resign with the same certificate as the test runner will be signed
+* Install
+
+### Appium test runner
+
+* `apktool d app-debug-androidTest.apk`
+* edit `AndroidManifest.apk` to set the instrumentation `android:targetPackage` to the package for the AUT
```
Comment (jlipps) [inline research.md]: is this an apk file or an xml file?

File: research.md [hunk 2]
```diff
@@ -188,3 +188,48 @@ How to run an espresso testcase from the command line:
...
+mInstrumentation.startActivitySync(intent);
+
+Activity mCurrentActivity = mInstrumentation.waitForMonitor(mSessionMonitor);
+assertNotNull(mCurrentActivity);
+
+```
+
+### Running the test
+
+`adb shell am instrument -w -r   -e debug false io.appium.espresso.BasicSample.test/android.support.test.runner.AndroidJUnitRunner`
```
Comment (jlipps) [inline research.md]: do away with extra spaces here?

File: research.md (hunk 1 above)
Comment (imurchie) [inline research.md]: XML, natürlich.

**Comment by imurchie** [comment]
Updated.

**Pull request merged**